packages/roosterjs-content-model-dom/lib/index.ts

```diff
@@ -15,7 +15,11 @@ export { areSameFormats } from './domToModel/utils/areSameFormats';
 export { isBlockElement } from './domToModel/utils/isBlockElement';
 export { buildSelectionMarker } from './domToModel/utils/buildSelectionMarker';
 
-export { updateMetadata, hasMetadata } from './modelApi/metadata/updateMetadata';
+export {
+    updateMetadata,
+    hasMetadata,
+    EditingInfoDatasetName,
```

Comment:
I'm trying not expose this dataset name actually to avoid developer's code access/modify metadata directly.